Assignment1/retirement.py

```diff
@@ -1,10 +1,38 @@
-class Retirement:
-    def __init__(self):
-        pass
+def year_can_retire(age, salary, saving, goal):
```

Comment:
Might want to rename "saving" to "savingPercent" or something that makes it clear that this value is a percentage. Even though I knew it was a percentage, my brain kept wanting to swap in "savings" instead.